src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -766,15 +875,107 @@ public static void runActionFromSearchEverywherePanel(RemoteRobot remoteRobot, S
     }
 
     /**
-     * Returns the RemoteText object representing the text found in the search everywhere output panel, or
+     * Selects the specified project from the list of detected projects shown in the 'Add Liberty project'
+     * dialog.
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     * @param projectName The name of the project to select.
+     */
+    public static void selectProjectFromAddLibertyProjectDialog(RemoteRobot remoteRobot, String projectName) {
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        DialogFixture addProjectDialog = projectFrame.find(DialogFixture.class,
+                DialogFixture.byTitle("Add Liberty project"),
+                Duration.ofSeconds(10));
+        JButtonFixture jbf = addProjectDialog.getBasicArrowButton();
+        jbf.click();
+
+        RemoteText remoteProject = findTextInListOutputPanel(addProjectDialog, projectName);
+        remoteProject.click();
+        JButtonFixture okButton = addProjectDialog.getButton("OK");
+        okButton.click();
+    }
+
+    /**
+     * Selects the specified project from the list of detected projects shown in the 'Remove Liberty project'
+     * dialog.
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     * @param projectName The name of the project to select.
+     */
+    public static void selectProjectFromRemoveLibertyProjectDialog(RemoteRobot remoteRobot, String projectName) {
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        DialogFixture addProjectDialog = projectFrame.find(DialogFixture.class,
```

Comment:
Do you mean combine the two methods you commented on into just one with the name of removeProjectDialog? 
I could do it, but for now, I purposely decided to keep these two actions (selection and response) separate because future tests may end up responding differently. If in the end it turns out, we don't need it, we could combine them.